pcmanfm/desktopwindow.cpp

```diff
@@ -948,7 +948,7 @@ bool DesktopWindow::eventFilter(QObject* watched, QEvent* event) {
             break;
         }
     }
-    return false;
+    return Fm::FolderView::eventFilter(watched, event);
```

Comment:
"Protection", "clean", "robust" and fear of future bugs become very subjective concepts when it comes to hairsplitting. We could discuss about our likes and dislikes for hours and reach no conclusion because there is nothing to conclude. In contrast, discussing about objective facts is always fruitful, even if long. You and I have had fruitful discussions -- and I hope we'll have many in future -- but this isn't one of them.